src/log.c

```diff
@@ -177,24 +113,27 @@ static int readHeader(Log *log)
     log->snapshot_last_idx = snapshot_last_index;
     log->index = snapshot_last_index;
 
+    /* validate CRC, by rebuilding header */
```

Comment:
Suggest doing a validation separately, as in readEntry()

Reply:
what do you mean?

Reply:
ok, I think I do that now